Update hacker-guide to clarify assignee logistics.md

veena-vee989 opened the pull request:
Trying to make it clear that hackers shouldn't be commenting on issues and asking to be added as assignee. 

Also trying to make it clear that they don't need to be assigned to start a PR.

@natestemen could you review and make sure these additions make sense? (I thought our rules were already clear, but maybe not.)

kel85uk commented:
@crazy4pi314 Hi Sarah, I was wondering if you had a chance to arrange the bank account number so I can bank in the agreed amount for donation? Thanks.

crazy4pi314 commented:
Yup sent you an email!

crazy4pi314 reviewed (approved):
LGTM

veena-vee989 committed:
Update hacker-guide to clarify assignee logistics.md

Trying to make it clear that hackers shouldn't be commenting on issues and asking to be added as assignee. 

Also trying to make it clear that they don't need to be assigned to start a PR.

@natestemen could you review and make sure these additions make sense? (I thought our rules were already clear, but maybe not.)

natestemen commented on src/pages/hacker-guide.md:
```diff
@@ -60,16 +60,18 @@ The main place that will be most helpful to you while you hack is connecting wit
-If you want to team up with other hackers on an issue that you have found, you can make a comment on the GitHub issue. Even if you don't plan to team up, commenting on GitHub may help other hackers know if someone else is working on that issue (that's fine, but depending on the issue, only one pull request may be merged to solve an issue). We ask that you not comment on more than 3-4 projects per ...
+If you want to team up with other hackers on an issue that you have found, you can make a comment on the GitHub issue. Even if you don't plan to team up, commenting on GitHub may help other hackers know if someone else is working on that issue (that's fine, but depending on the issue, only one pull request may be merged to solve an issue). We ask that you not comment on more than 3-4 projects per ...
+
+**Note**: You do not need to comment on an issue to start a Pull Request. You can immediately start working on it. If a maintainer decides that your solution is the best for their issue, they will accept your pull request, assign you to the issue, and close then close it. **Please do not comment on projects and ask to be added as an assignee before you create a PR.**
```
```suggestion
**Note**: You do not need to comment on an issue to start a Pull Request. You can immediately start working on it. If a maintainer decides that your solution is the best for their issue, they will accept your pull request, assign you to the issue, and close it. **Please do not comment on projects and ask to be added as an assignee before you create a PR.**
```

natestemen commented on src/pages/hacker-guide.md:
```diff
@@ -60,16 +60,18 @@ The main place that will be most helpful to you while you hack is connecting wit
 Once you have a good solution to the issue, and have checked out the project's contributing guide (if they have one), you are ready to make your pull request! Check out the guides above if you are not familiar with `git` or GitHub, or if you run into issues making the contribution, just ask on the [Unitary Foundation discord](http://discord.unitary.foundation/)!
 
-We ask that you not create more than 4 pull requests at one time. This is to ensure that you are not overwhelming yourself, that other hackers are getting a chance to contribute, and that our maintainers are not inundated with PRs that aren’t fully complete. 
+We ask that you not have more than 4 pull requests open at one time across all projects. This is to ensure that you are not overwhelming yourself, that other hackers are getting a chance to contribute, and that our maintainers are not inundated with PRs that aren’t fully complete. 
```
```suggestion
We ask that you not have more than 4 pull requests open at one time across all unitaryHACK projects. This is to ensure that you are not overwhelming yourself, that other hackers are getting a chance to contribute, and that our maintainers are not inundated with PRs that aren’t fully complete. 
```

natestemen reviewed (approved):
Approving with two small recommendations.

veena-vee989 committed:
Update src/pages/hacker-guide.md

Co-authored-by: nate stemen <nate@unitary.foundation>

veena-vee989 committed:
Update src/pages/hacker-guide.md

Co-authored-by: nate stemen <nate@unitary.foundation>
